README.md

```diff
@@ -60,7 +60,7 @@ This can be done using the ``GurobiSolver`` object. Here is how to create a *JuM
 ```julia
 using JuMP, Gurobi
 
-m = Model(solver=GurobiSolver(Presolve=0))
+model = Model(with_optimizer(Gurobi.Optimizer))
```

Comment:
Can you keep the presolve option example please?

Reply:
Done.